cli/module_loader.rs

```diff
@@ -27,7 +27,7 @@ use crate::node;
 use crate::node::CliNodeCodeTranslator;
 use crate::npm::CliNpmResolver;
 use crate::resolver::CjsTracker;
-use crate::resolver::CliNodeResolver;
+use crate::resolver::CliNpmReqResolver;
```

Comment:
I know it seems a bit crazy to have all these different little structs, but it will make our code more versatile in the future.